### PR TITLE
adding uwhoisd installation

### PR DESCRIPTION
**Pull request by adrima01**
Pull requests should be opened against the `main` branch. For more information on contributing to Lookyloo documentation, see the [Contributor Guidelines](https://www.lookyloo.eu/docs/main/contributor-guide.html).

## Type of change

**Description:**
Adding uwhoisd installation and configuration so that the tests can pass in the github actions.

**Select the type of change(s) made in this pull request:**
- [ ] Bug fix *(non-breaking change which fixes an issue)*
- [x] New feature *(non-breaking change which adds functionality)*
- [ ] Documentation *(change or fix to documentation)*

---------------------------------------------------------------------------------------------------------

Fixes #issue-number

## Proposed changes 

*
*
*

### CODE REVIEW

File: .github/workflows/instance_test.yml [hunk 1]
```diff
@@ -59,6 +75,7 @@ jobs:
           poetry run tools/validate_config_files.py --check
           poetry run tools/validate_config_files.py --update
           poetry run tools/3rdparty.py
+          jq '.UniversalWhois.enabled = true' config/modules.json > temp.json && mv temp.json config/modules.json
```
Comment (Rafiot) [inline .github/workflows/instance_test.yml:78]: nice one! I didn't know you could use jq that way.

**Comment by Rafiot** [comment]
Oh, the test will never finish: you cloned uwhoisd in the lookyloo repository, but it expects find redis in the parent directory and it's not there.

You will need to do something like that: https://github.com/Lookyloo/lookyloo/blob/main/.github/workflows/instance_test.yml#L36

**Comment by adrima01** [comment]
> Oh, the test will never finish: you cloned uwhoisd in the lookyloo repository, but it expects find redis in the parent directory and it's not there.
> 
> You will need to do something like that: https://github.com/Lookyloo/lookyloo/blob/main/.github/workflows/instance_test.yml#L36
> 
> ... and you just did it ;)

seems like it still won't work

**Comment by Rafiot** [comment]
Thanks!

**Pull request merged**